<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractParametersDialog</class>
 <widget class="QDialog" name="ExtractParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extraction Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Location</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="4">
       <widget class="QCheckBox" name="cbCreateSubfolder">
        <property name="text">
         <string>Create subfolder: ARTIST/ALBUM</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="eOutputDirectory">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="bBrowseDir">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="IconWidget" name="iwFolder" native="true">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>CD Extraction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QRadioButton" name="rbParanoiaExtract">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ClickableBuddyLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;b&gt;&amp;Fast Mode&lt;/b&gt;&lt;br&gt;
Trust the hardware to read data correctly, which almost all drives do.</string>
        </property>
        <property name="buddy">
         <cstring>rbFastExtract</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbFastExtract">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="ClickableBuddyLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;b&gt;&amp;Paranoia Mode&lt;/b&gt;&lt;br&gt;Perform extra error detection and correction. This is very slow.</string>
        </property>
        <property name="buddy">
         <cstring>rbParanoiaExtract</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="IconWidget" name="iwCd" native="true">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Audio File Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QRadioButton" name="rbFlac">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="ClickableBuddyLabel" name="lMp3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;b&gt;&amp;MP3&lt;/b&gt;, 192kbit/s (.mp3)&lt;br&gt;
Lossy compression with potentially audible quality degradation.</string>
        </property>
        <property name="buddy">
         <cstring>rbMp3</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbMp3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="ClickableBuddyLabel" name="label_3">
        <property name="text">
         <string>&lt;b&gt;F&amp;LAC&lt;/b&gt; (.flac)&lt;br&gt;
Lossless compression. Recommended, but not all players can handle it.</string>
        </property>
        <property name="buddy">
         <cstring>rbFlac</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="ClickableBuddyLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;b&gt;RIFF &amp;WAVE&lt;/b&gt; (.wav)&lt;br&gt;
Uncompressed file format without metadata. Not recommended.</string>
        </property>
        <property name="buddy">
         <cstring>rbWav</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rbWav">
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="IconWidget" name="iwFileFormat" native="true">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bStart">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Begin Extraction</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableBuddyLabel</class>
   <extends>QLabel</extends>
   <header>uiutil/clickablebuddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>uiutil/iconwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>eOutputDirectory</tabstop>
  <tabstop>bBrowseDir</tabstop>
  <tabstop>cbCreateSubfolder</tabstop>
  <tabstop>rbFastExtract</tabstop>
  <tabstop>rbParanoiaExtract</tabstop>
  <tabstop>rbFlac</tabstop>
  <tabstop>rbMp3</tabstop>
  <tabstop>rbWav</tabstop>
  <tabstop>bCancel</tabstop>
  <tabstop>bStart</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExtractParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>1200</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bStart</sender>
   <signal>clicked()</signal>
   <receiver>ExtractParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
